controls/SV-257804.rb

```diff
@@ -32,4 +32,17 @@
     it { should be_disabled }
     it { should be_blacklisted }
   end
+
+  config_files = command('find /etc/modprobe.conf /etc/modprobe.d/* -print0').stdout.split("\0")
```

Comment:
Does the `kernel_module` resource not do this for you already?

Reply:
https://github.com/inspec/inspec/blob/main/lib/inspec/resources/kernel_module.rb

Reply:
It appears so. I was under the impression that `kernel_module` was testing the results of the configuration being loaded (maybe using something `lsmod` or `modprobe` command itself) rather than directly looking at how it was configured. In this case, this means that this whole PR is redundant, right?